src/Sarif/Sarif.csproj

```diff
@@ -39,6 +39,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
It's claiming it's a binary file because VS apparently generates this file as UTF-16 and GitHub can't deal. @jericahuang has fixed this problem once before by re-saving the file as UTF-8 (no signature), and I think @kschecht has done the same. This file shouldn't be changed/added in @Rolstenhouse's branch at all, though.

You might have to just save the new files (which you know now work) off to the side, abandon this branch, check out master, and add the new files to master. Your branch seems to be messed up somehow (as we see by the huge number of commits in this change.